djstripe/models/core.py

```diff
@@ -762,6 +762,36 @@ def create(cls, subscriber, idempotency_key=None, stripe_account=None):
 
         return customer
 
+    @classmethod
```

Comment:
I'm not sure if this is still the case, but we don't have this method on Customer because customers should always originate from a local object. Can this logic be placed in the above `create` method?

Reply:
Can you elaborate on what you mean that customers should originate from a local object? I'm not sure I understand.

Right now the customer model inherits this method from `StripeModel`. The reason I've made this happen here is that this is the method that's called in [`sync_from_stripe_data`](https://github.com/dj-stripe/dj-stripe/blob/dbd4b1caa55c2b9d49111ed4f32b25e5342a06a3/djstripe/models/base.py#L866). I've needed to expand the method for Customer to check if there are tax_ids in the payload and create the TaxId object locally if it doesn't exist. It's only doing that as an addition to calling the original method (line 779). As to adding this to `create`, it's also possible to add TaxIds to existing customers. In that case you'd skip the TaxId on the customer sync since the `create` method wouldn't get called.

Reply:
@Tuuleh yep, you're totally right. I was too tired when I was reviewing this :)